src/app/projects/pages/projects-list/projects-list.component.html

```diff
@@ -200,6 +200,36 @@
                 </li>
               </ul>
             </li>
+            <li *ngIf="project.cellXGeneLinks.length">
+              cellxgene: <br />
+              <ul class="comma-list">
+                <li *ngFor="let link of project.cellXGeneLinks">
+                  <a class="vf-link" href="{{ link.href }}" target="_blank">{{
+                    link.name
+                  }}</a>
+                </li>
+              </ul>
+            </li>
```

Comment:
Have you considered making a component out of this "list of links"?

Reply:
I have not

Reply:
Is this what you had in mind?
``` projects-list.component.html
<li *ngIf="project.enaAccessions.length">
  <list-of-links
    name="ENA"
    [links]="project.enaAccessions"
  ></list-of-links>
</li>
```